.github/workflows/gitleaks.yaml

```diff
@@ -69,5 +69,14 @@ jobs:
         with:
           configuration-data: |
             threshold: 0.95
+            ignore: |
+              .git
+              man/figures/*.svg
```

Comment:
could you change it to **/*.svg ?
could you change it to **/*.png ?

Reply:
`**/*.png` and `**/*.svg`